en/mozillavpn.xliff

```diff
@@ -0,0 +1,556 @@
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file datatype="plaintext" original="../src/ui/components/VPNCheckBoxAlert.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.turnOffAlert.enabling">
+      <source xml:space="preserve">VPN must be off before enabling</source>
+      <context-group purpose="location"><context context-type="linenumber">32</context></context-group>
+      <note annotates="source" from="developer">This is a Setting which requires the VPN to be disconnected to change state</note>
+    </trans-unit>
+    <trans-unit id="vpn.turnOffAlert.disabling">
+      <source xml:space="preserve">VPN must be off before disabling</source>
+      <context-group purpose="location"><context context-type="linenumber">35</context></context-group>
+      <note annotates="source" from="developer">This is a Setting which requires the VPN to be disconnected to change state</note>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/components/VPNConnectionInfo.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.connectionInfo.ip">
+      <source xml:space="preserve">IP: %1</source>
+      <context-group purpose="location"><context context-type="linenumber">118</context></context-group>
+      <note annotates="source" from="developer">The Current Ip adress</note>
+    </trans-unit>
+    <trans-unit id="vpn.connectionInfo.download">
+      <source xml:space="preserve">Download</source>
+      <context-group purpose="location"><context context-type="linenumber">130</context></context-group>
+      <note annotates="source" from="developer">the current download speed. The speed is shown at the next line.</note>
```

Comment:
nit: **on** the next line

For consistency, let's start with an uppercase letter

> The current download speed. The speed is shown on the next line.